rollup.config.js

```diff
@@ -0,0 +1,119 @@
+/**
+ * Shared config used by individual packages' Rollup configs
+ */
+
+import license from 'rollup-plugin-license';
+import resolve from '@rollup/plugin-node-resolve';
+import replace from '@rollup/plugin-replace';
+import { terser } from 'rollup-plugin-terser';
+import typescript from 'rollup-plugin-typescript2';
+
+export const paths = {
+  '@sentry/browser': ['../browser/src'],
+  '@sentry/core': ['../core/src'],
+  '@sentry/hub': ['../hub/src'],
+  '@sentry/minimal': ['../minimal/src'],
+  '@sentry/types': ['../types/src'],
+  '@sentry/utils': ['../utils/src'],
+};
+
+/**
+ * Create a plugin to add an identification banner to the top of stand-alone bundles.
+ *
+ * @param title The title to use for the SDK, if not the package name
+ * @returns An instance of the `rollup-plugin-license` plugin
+ */
+export function makeLicensePlugin(title) {
```

Comment:
Unit test the plugins themselves, or unit test the code generating the plugins?

The former would be kinda like the treeshaking scenarios - do we get the bundle we expect to get by including x, y, or z plugin? The latter would be basically snapshot testing against hardcoded configs, which we certainly could do, and isn't a bad idea, actually. I think it doesn't make sense for this PR, because there's not much code involved, but upcoming PRs might be good candidates.

Reply:
Cool, I was referring to the latter - but I'm fine with holding off on it!